index.html

```diff
@@ -109,6 +109,7 @@ <h3>Your decrypted orders:</h3>
             <pre id="logger"></pre>
             <div id="loadbar"></div>
         </main>
+        <script src="https://cdn.jsdelivr.net/npm/es6-promise@4/dist/es6-promise.auto.min.js"></script>
```

Comment:
We should be able to delete this as we are importing `es6-promises` as a node module.